go.sum

```diff
@@ -1,16 +1,11 @@
-github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869/go.mod h1:Ekp36dRnpXw/yCqJaO+ZrUyxD+3VXMFFr56k5XYrpB4=
```

Comment:
This isn't even the right `assert` library. What happens sometimes is that I write some test code using `assert.` and the IDE helpfully goes "oh, you haven't imported that package yet, let me find one that's called 'assert'" and adds an import for the wrong one, and then it gets into `go.sum` and stays there indefinitely.